### ISSUE TITLE
[BUG] main site should not be installable

### ISSUE THREAD

**Issue by KenEucker** [KenEucker/biketag-vue#250]
**Describe the bug**
The "Install to home screen" option in browsers should not appear for the main site. This should only appear for games.

**To Reproduce**
1. Go to main site.
2. See "install to homescreen" option
3. install app

**Expected behavior**
This shouldn't happen for the main site.

**Screenshots**

**Comment by zatr** [comment]
Please define what you mean by 'main site'

If it's this https://www.biketag.org/ then I don't see any "install to homescreen" option.

Maybe this bug is already resolved.

**Comment by KenEucker** [comment]
@zatr , yes, BikeTag.org is what I mean by "main site".  I didn't currently see the "ready to install" modal on the site when I visit with my phone but I'm not yet ready to call this ticket closed. I'll look into it a little more when I'm back to my computer. 

Thanks for following up!

**Comment by KenEucker** [comment]
Confirmed this is no longer an issue.
